SPECS/xha.spec

```diff
@@ -1,15 +1,12 @@
+%global package_speccommit f7bea5d97a5d59bee0a3ca6483a7efa0c7115842
+%global package_srccommit v10.3.1
 Summary: xha - XenServer proprietary HA daemon
 Name:    xha
-Version: 10.1.0
-Release: 2.1%{?dist}
+Version: 10.3.1
+Release: 3.1%{?xsrel}%{?dist}
 License: GPLv2
 URL:     https://github.com/xenserver/xha
-
-Source0: https://code.citrite.net/rest/archive/latest/projects/XSU/repos/xha/archive?at=v10.1.0&prefix=xha-10.1.0&format=tar.gz#/xha-10.1.0.tar.gz
-
-
-Provides: gitsha(https://code.citrite.net/rest/archive/latest/projects/XSU/repos/xha/archive?at=v10.1.0&prefix=xha-10.1.0&format=tar.gz#/xha-10.1.0.tar.gz) = 20b7e5244f9e25b199ea8e3ef5b935f4e9b4247d
-
+Source0: xha-10.3.1.tar.gz
```

Comment:
Our Patch1000 is still needed, I presume?

Reply:
In theory no because we use http-nbd-transfer with NBD volumes, in practice we have not tested without it and it can be useful for debugging. May be something to evaluate later. But there's no risk in leaving it either.